agent/api.go

```diff
@@ -177,13 +177,17 @@ func (a *API) SetChatAccess(id string, access objects.Access) error {
 
 // TransferChat transfers chat to agent or group.
 func (a *API) TransferChat(chatID, targetType string, ids []interface{}, force bool) error {
-	return a.Call("transfer_chat", &transferChatRequest{
-		ChatID: chatID,
-		Target: &transferTarget{
+	var target *transferTarget
+	if targetType != "" || len(ids) > 0 {
```

Comment:
opt: I'm wondering if we can put some validation here (as if target is present, both of those are mandatory),  but afair we don't do such validation anywhere in SDK so maybe it's worth creating separate improvement task

Reply:
Ok, we have discussed it and decided that validation will be added in separate, minor version (not patch) and it will validate all methods.